<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_new_project</class>
 <widget class="QWidget" name="add_new_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>170</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="add_programming_button">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>Add programming project</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_everyday_button">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>Add everyday project</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_button">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>Return to main menu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_button_projects">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>Exit program</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
